extensions/amp-story/1.0/animation.js

```diff
@@ -541,13 +578,42 @@ export class AnimationManager {
     );
   }
 
+  /**
+   * Gets animation definitions.
+   * These are either from an <amp-animation> config (WebAnimationDefs), or
+   * resolved from presets via animate-in attributes (StoryAnimationDefs).
+   * If a root element contains both kinds of definitions, they'll run
+   * concurrently.
+   * @return {!Array<!WebAnimationDef|!StoryAnimationDef>}
+   * @private
+   */
+  getAnimationDefs_() {
+    // From preset
+    const animationDefs = Array.prototype.map.call(
+      scopedQuerySelectorAll(this.root_, ANIMATABLE_ELEMENTS_SELECTOR),
+      (el) => this.createAnimationDef(el, this.getPreset_(el))
+    );
+    // <amp-animation>
+    const ampAnimationElement = childElementByTag(this.root_, 'amp-animation');
+    if (ampAnimationElement) {
+      animationDefs.push(parseAmpAnimationConfig(ampAnimationElement));
```

Comment:
/cc @gmajoulet 

So given other comments here, such as `<amp-animation animate-in-after>` and so on, I am actually starting to warm come quite a bit to the idea of `<amp-story-animation>`. It seems to be gaining more and more of its own identity here. And if we want to allow just `amp-animation` later, we, of course, still could do that via deprecation path.

Reply:
To simplify this PR, I'll just use `amp-story-animation` as an element without implementation and remove `amp-animation` changes.